etrago/tools/plot.py

```diff
@@ -2703,7 +2717,6 @@ def plot_grid(
             flow=flow,
             title=title,
             geomap=False,
-            boundaries=geographical_boundaries,
```

Comment:
Removing this line created some problems because the defined geographical boundaries were not used anymore. I fixed this in my latest commit.